### DISCUSSION TITLE
Build parameter

### DISCUSSION THREAD

**Discussion by whc10002** [microsoft/onnxruntime#13742] [Other Q&A]
I want to know which parameters are used in the shared library of onnxruntime-linux-x64-1.13.1. Can anyone provide it?

**Comment by tangjicheng1** [comment]
I am not sure. Maybe this is what you want.
https://github.com/microsoft/onnxruntime/blob/4ca62b9ee827d243d1b1356126a033cf8710b25a/.github/workflows/linux.yml#L28

**Reply by whc10002** [reply]
I will have try to make sure the output library is the same size as the download from the official site. Many thanks.
